client/reader/lib/feed-display-helper/index.js

```diff
@@ -58,4 +43,4 @@ const exported = {
 
 export default exported;
 
-export const { formatUrlForDisplay, getFeedTitle, getFeedStreamUrl, getSiteUrl } = exported;
```

Comment:
Let's convert this module to a normal ES module, i.e., remove the CJS-compat `exported` object. It's not used anywhere, all imports are normal named ES imports.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/86996/commits/d50632d25ac4c4d161bf74a8ccca5e385fd981bb